etna/core/mixins.py

```diff
@@ -89,6 +90,50 @@ def to_dict(self):
         params["_target_"] = BaseMixin._get_target_from_class(self)
         return params
 
+    def _update_nested_dict_with_flat_dict(self, params_dict: dict, flat_dict: dict):
+        """Change flat specification into dict of nested params.
+
+        Parameters
+        ----------
+        **params_dict: dict
+            dict with nested parameters structure, e.g {"model": {"learning_rate": value1}}
```

Comment:
We should wrap {"model": {"learning_rate": value1}} like this:
```
``{"model": {"learning_rate": value1}}``
```
Otherwise, it will look bad in online documentation after rendering.